package io.github.devandref.ms_avaliador_credito.application.exception;

public class DadosClienteNotFoundException extends Exception{

    public DadosClienteNotFoundException() {
        super("Dados do cliente n√£o encontrados para o CPF informado.");
    }

}
